packages/frontend/src/components/Button/style.module.scss

```diff
@@ -36,10 +35,12 @@
   &.secondary {
     background-color: var(--buttonSecondaryBackground);
     color: var(--buttonSecondaryText);
+    border: 1px solid #ccc;
   }
 
   &.danger {
-    color: var(--buttonDangerText);
+    color: var(--buttonPrimaryText);
+    background-color: var(--buttonDangerText);
   }
```

Comment:
The contrast of the danger and the primary button is low

![image](https://github.com/user-attachments/assets/5b296603-65d3-4489-b13f-1bef2fd7fbcf)

![image](https://github.com/user-attachments/assets/23d0d05b-24a5-4eba-9eb6-95bea1cbed86)

Reply:
Can you suggest a color theme here, which has better contrast?

Reply:
In this particular case, I think a close button at the top would make more sense.
But perhaps this is for a separate merge request.

Reply:
going up to white, seems reasonable. the color of the button, however should stay. these colors are used widely in a lot of themes, including apple, they cannot be all wrong :)

Reply:
The contrast is still not satisfactory.

![image](https://github.com/user-attachments/assets/03ba7ff5-922c-419e-929d-fb6ed723bdfc)

It used to be better, on dark theme at least:

![image](https://github.com/user-attachments/assets/7385002e-0869-4716-beb2-e152558946ba)